draft-foudil-securitytxt.md

```diff
@@ -257,11 +260,12 @@ implementors MUST ignore any fields they do not explicitly support.
 # File Format Description
 
 The expected file format of the security.txt file is plain text as defined
```

Comment:
Wondering also if we want to change "plain text" to read "**plain text (MIME type "text/plain")**"